src/js/components/day.jsx

```diff
@@ -52,11 +59,15 @@ export default class Day extends React.Component {
           description += ' - ';
         }
       }
+      let timeslotDate = momentTime.clone();
```

Comment:
Moment.js does not provide an **immutable** object. Hence, the object must be cloned unless the change to the original time is desired.